app/views/custom_fields/_tab.html.erb

```diff
@@ -27,6 +27,30 @@ See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
 
+<% if tab[:name] == 'ProjectCustomField' %>
```

Comment:
On both pages, the teaser is now shown twice:
<img width="700" alt="Bildschirmfoto 2022-08-08 um 10 41 17" src="https://user-images.githubusercontent.com/7457313/183379294-807a12c3-23cb-423b-84d9-9e659569c882.png">